mobile/src/main/java/rs/readahead/washington/mobile/views/settings/SecuritySettings.kt

```diff
@@ -240,7 +240,7 @@ class SecuritySettings : BaseFragment() {
         intent!!.putExtra(RETURN_ACTIVITY, returnCall.getActivityOrder())
         intent.putExtra(IS_FROM_SETTINGS, true)
         startActivity(intent)
-        activity.finish()
+        //activity.finish()
```

Comment:
I think if we just want the user to came back to the security settings we need just not finish the current activity..